windows/include/liblouis.h

```diff
@@ -44,7 +44,10 @@ extern "C" {
 typedef WIDECHAR_TYPE widechar;
 typedef unsigned short formtype;
 
-#ifdef _WIN32

```

Comment:
Why is this changed? This code path will never be reached, as this header file will only be used by a windows developer tyring to build liblouis either with nmake or Visual Studio.

Is it just so that synchronization between the two header files is easier?